src/scripts/content/teamwork.js

```diff
@@ -9,7 +9,7 @@ togglbutton.render('div.taskRHS:not(.toggl), div.row-rightElements:not(.toggl)',
     isTKO = false,
     className = 'huh',
     container = $('.taskIcons', elem),
-    project = $("#projectName > span").textContent;
+    project = $(".w-header-titles__project-name a").textContent;
```

Comment:
Hey Neyato,
Thanks for contributing.
This apparently needs a `trim()` otherwise it won't match the project names on Toggl, because of the extra spaces.

Reply:
Ok thanks for the information. I changed it now.